lib/spack/spack/schema/modules.py

```diff
@@ -257,4 +320,7 @@ def update(data):
         changed = True
         data['default'] = default
 
+    # translate exclude/include to blacklist/whitelist
```

Comment:
I think this comment should be backward:
```suggestion
    # translate blacklist/whitelist to exclude/include
```
right?

Reply:
fixed